contents/swift/welcome-to-swift.md

```diff
@@ -312,7 +350,26 @@ Reference cycles are bad, because they cause memory leaks. Even though A and B a
 
 Here is an [article](https://krakendev.io/blog/weak-and-unowned-references-in-swift) with greater in-depth explanation and examples.
 
-## CocoaPods
+### Benefit: Reuse Existing Code
+
+When you have been working on a Swift project for a while and start to think "Hmm... Someone has probably done something like this before" or "This is a common problem and there should be a library for this", check out [CocoaPods](https://guides.cocoapods.org/using/getting-started.html), the package/library manager for Swift. Apart from using existing library, you may also make your code as a "library" by CocoaPods.
+
+## How to Get Started
```

Comment:
Have a read through this section one more time!

The gist is clear and understandable. However, there are places where the phrasing/grammer can be improved. 

e.g. "If you did not learn any other programming languages before" => "If you have not learnt any other programming languages before"

Reply:
If you have done something new with Swift, you can also make your code into a library with CocoaPods for others to use!